testsuite/features/secondary/trad_openscap_audit.feature

```diff
@@ -34,19 +40,24 @@ Feature: openSCAP audit of traditional client
     And I click on "Update Organization"
     Then I should see a "Organization SUSE Test was successfully updated." text
 
-  Scenario: Delete audit results from traditional client
+  Scenario: Cleanup: delete audit results from traditional client
     Given I am on the Systems overview page of this "sle_client"
     When I follow "Audit" in the content area
     And I follow "List Scans" in the content area
     And I click on "Select All"
     And I click on "Remove Selected Scans"
     And I click on "Confirm"
-    Then I should see a "deleted. 0 SCAP Scan(s) retained" text
+    Then I should see a "1 SCAP Scan(s) deleted. 0 SCAP Scan(s) retained" text
```

Comment:
These kinds of assumptions are very problematic for idempotency and it's a source of issues if we decide to run the tests in parallel. Please keep that in mind.

Reply:
Agree. Will remove it everywhere. Was done to align, but I aligned the wrong way round.

Reply:
Done.

Reply:
As a side note, it could not have created problems with parallelism, as it happens on different clients, and each has its own audit screen. But OK.